### PR TITLE
Show an error message when compile response is empty

### PR DESCRIPTION
**Pull request by jarroyo3**

### CODE REVIEW

**Comment by jarroyo3** [comment]
Hi @sahildua2305 

Do I correctly now? :)

File: hackIDE/static/hackIDE/js/custom.js [hunk 1]
```diff
@@ -269,7 +269,13 @@ $(document).ready(function(){
 						$(".output-io-info").hide();
 						$(".compile-status").children(".value").html("--");
 						$(".error-key").html("Compile error");
-						$(".error-message").html(response.compile_status);
+						
+						var compileMsgResponse = response.compile_status;
+						if (response.compile_status == "" || response.compile_status.length <= 1) {
+							compileMsgResponse = "An error ocurred while compiling your code or your code is not correct";
```
Comment (sahildua2305) [inline hackIDE/static/hackIDE/js/custom.js]: We can make this explicit for the end user when we receive empty compile response. Wouldn't that be a better scenario? If so, we can change it to something like `"Empty Compile Response. Something went wrong while compiling"`.

File: hackIDE/static/hackIDE/js/custom.js [hunk 2]
```diff
@@ -269,10 +269,10 @@ $(document).ready(function(){
 						$(".output-io-info").hide();
 						$(".compile-status").children(".value").html("--");
 						$(".error-key").html("Compile error");
-						
+
 						var compileMsgResponse = response.compile_status;
 						if (response.compile_status == "" || response.compile_status.length <= 1) {
-							compileMsgResponse = "An error ocurred while compiling your code or your code is not correct";
+							compileMsgResponse = "Empty Compile Response. Something went wrong while compiling";
```
Comment (sahildua2305) [inline hackIDE/static/hackIDE/js/custom.js]: Looks good. Minor nit - let's end the message with a '.'?

**Comment by sahildua2305** [comment]
Thanks a lot @jarroyo3, merging...

**Pull request merged**